web-marketplace/src/components/organisms/Collaborators/RoleDropdown.tsx

```diff
@@ -0,0 +1,276 @@
+import { useEffect, useRef, useState } from 'react';
+import { msg } from '@lingui/macro';
+import { useLingui } from '@lingui/react';
+
+import CheckIcon from 'web-components/src/components/icons/CheckIcon';
+import DropdownIcon from 'web-components/src/components/icons/DropdownIcon';
+import InfoTooltip from 'web-components/src/components/tooltip/InfoTooltip';
+import { cn } from 'web-components/src/utils/styles/cn';
+
+import {
+  ORG_ADMIN,
+  ORG_EDITOR,
+  ORG_MEMBER_SETTINGS,
+  TOOLTIP_EXTERNAL_ADMIN,
+  TOOLTIP_ONLY_ADMIN,
+  TOOLTIP_ROLE,
+} from './Collaborators.constants';
+import { ProjectRoleType, RoleDropdownProps } from './Collaborators.types';
+import { ROLE_HIERARCHY, ROLE_OPTIONS } from './Collaborators.utils';
+
+export const RoleDropdown = ({
```

Comment:
we already have some `web-components/src/components/inputs/new/SelectTextField` component which could have been reused I think, instead of rebuilding the dropdown behavior from scratch

Reply:
did you look into that?